client/server/pages/index.js

```diff
@@ -52,24 +52,20 @@ const debug = debugFactory( 'calypso:pages' );
 
 const calypsoEnv = config( 'env_id' );
 
-let branchName;
 function getCurrentBranchName() {
-	if ( ! branchName ) {
-		try {
-			branchName = execSync( 'git rev-parse --abbrev-ref HEAD' ).toString().replace( /\s/gm, '' );
-		} catch ( err ) {}
+	try {
+		return execSync( 'git rev-parse --abbrev-ref HEAD' ).toString().replace( /\s/gm, '' );
+	} catch ( err ) {
+		return undefined;
 	}
-	return branchName;
 }
 
-let commitChecksum;
 function getCurrentCommitShortChecksum() {
-	if ( ! commitChecksum ) {
-		try {
-			commitChecksum = execSync( 'git rev-parse --short HEAD' ).toString().replace( /\s/gm, '' );
-		} catch ( err ) {}
+	try {
```

Comment:
Hm, why did we revert these?

Reply:
Great question -- I guess they got re-added during a rebase?